pkgs/tools/admin/ssmsh/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoModule, fetchurl }:
+
+buildGoModule rec {
+  pname = "ssmsh";
+  version = "1.4.3";
+
+  src = fetchurl {
```

Comment:
fetchFromGitHub can be used here right?

Reply:
I liked the simplicity of fetchurl, but yes you're right, it looks like most of the Go packages use fetchFromGithub, so I'll switch over to that...

Reply:
Ok switched over :+1: